complete/auth.js

```diff
@@ -13,10 +13,10 @@
 
 module.exports = function(config) {
 
-  var googleapis = require('googleapis');
+  var {google} = require('googleapis');
```

Comment:
The client library changed. The exported module is structure differently. It's good practice to use the  name that the module exports, hence the shorthand destructuring syntax works so nicely. Otherwise it would be `var {google: googleapis} = require('googleapis')`. And it's consistent with the samples in the `googleapis` documentation.